app/validators/case_details/answers_validator.rb

```diff
@@ -14,11 +16,9 @@ def initialize(record)
 
     def validate # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
       errors.add(:has_case_concluded, :blank) unless case_concluded_complete?
+      errors.add(:is_preorder_work_claimed, :blank) unless preorder_work_complete?
 
-      unless not_means_tested?
-        errors.add(:is_preorder_work_claimed, :blank) unless preorder_work_complete?
-        errors.add(:is_client_remanded, :blank) unless client_remanded_complete?
-      end
+      errors.add(:is_client_remanded, :blank) if !non_means_tested? && !client_remanded_complete?
```

Comment:
Rubocop made this impossible to read - happy to have a go at changing it to make it better if anyone keen for it